admin/tests/integration/components/sessions/jury-comment_test.js

```diff
@@ -3,30 +3,133 @@ import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 import moment from 'moment';
+import sinon from 'sinon';
+import clickByLabel from '../../../helpers/extended-ember-test-helpers/click-by-label';
+import fillInByLabel from '../../../helpers/extended-ember-test-helpers/fill-in-by-label';
 
 module('Integration | Component | Sessions::JuryComment', function(hooks) {
   setupRenderingTest(hooks);
 
-  test('it renders', async function(assert) {
-    // given
-    this.set('author', 'Vernon Sanders Law');
-    this.set('date', new Date('2021-06-21T14:30:21Z'));
-    this.set('comment', 'L\'expérience est un professeur cruel car elle vous fait passer l\'examen, avant de vous expliquer la leçon.');
-    const expectedDate = moment(this.date).format('DD/MM/YYYY à HH:mm');
-
-    // when
-    await render(hbs`
-      <Sessions::JuryComment
-        @author={{this.author}}
-        @date={{this.date}}
-        @comment={{this.comment}}
-      />
-    `);
-
-    // then
-    assert.contains('Commentaire de l\'équipe Certification');
-    assert.dom('.session-jury-comment__author').hasText('Vernon Sanders Law');
-    assert.dom('.session-jury-comment__date').hasText(expectedDate);
-    assert.dom('.session-jury-comment__content').hasText('L\'expérience est un professeur cruel car elle vous fait passer l\'examen, avant de vous expliquer la leçon.');
+  module('when there is no comment', function() {
+    test('it renders an empty comment form', async function(assert) {
+      // given
+      this.author = null;
+      this.date = null;
+      this.comment = null;
+
+      // when
+      await render(hbs`
+        <Sessions::JuryComment
+          @author={{this.author}}
+          @date={{this.date}}
+          @comment={{this.comment}}
+        />
+      `);
+
+      // then
+      assert.dom('[aria-label="Texte du commentaire"]').exists();
+      assert.dom('[aria-label="Texte du commentaire"]').hasValue('');
+      assert.dom('[data-test-id="jury-comment-save-button"]').exists();
+      assert.dom('[data-test-id="jury-comment-cancel-button"]').doesNotExist();
+    });
+
+    module('when the form is submitted', function() {
+      test('it saves the new comment', async function(assert) {
```

Comment:
```suggestion
      test('it saves the new comment', async function(assert) {
```
C'est preeeeesque borderline en vrai comme nommage (et je chipote un peu j'avoue).
Quand on se met du point de vue du component, il en a aucune idée que derrière ça provoque un appel API qui sauvegarde. 
Quand on s'intéresse froidement à quelle est sa vraie responsabilité, c'est de :
- Déclencher la fonction passée en argument à la construction (je ne sais pas ce qu'elle fait derrière)
- Passer en paramètre de la fonction le commentaire

C'est ça sa vraie responsabilité et ce qu'on doit tester ici et qui doit transparaître aussi dans le nommage du it()

Reply:
Du coup pour moi ça rentre un peu en conflit avec une autre règle qu'on évoque parfois (mais qui n'est pas gravée dans le Confluence pour l'instant) qui est que le texte du `it` devrait être compréhensible par quelqu'un du métier et donc ne pas décrire ce que fait techniquement le code. Ou bien est-ce que ce n'est vrai que pour les tests d'acceptance ?

Reply:
> qui est que le texte du it devrait être compréhensible par quelqu'un du métier et donc ne pas décrire ce que fait techniquement le code. Ou bien est-ce que ce n'est vrai que pour les tests d'acceptance ?

Ah, touchée. Il faut en discuter t'as raison.

Reply:
Pour les tests d'acceptence/E2E ca a du sens. Mais pour l'integration + unitaire, ca ne me semble pas pertinent d'étre compréhensible par qqun du metier, parce qu'on teste des "composants" techniques, soit unitairement, soit en interaction avec un autre. Ca reste technique.
Je pense que le besoin d'avoir qque chose de lisible, c'est l'enchainement des describe/it, mais pas forcement par le métier